docs/3-reference/useCheckoutPricingReturn.stories.mdx

```diff
@@ -0,0 +1,63 @@
+import { Subtitle } from '@storybook/addon-docs/blocks';
+import { PropsTable } from '@storybook/components';
+
+<Meta title="Reference/UseCheckoutPricingReturn" />
+
+# UseCheckoutPricingReturn
+
+<Subtitle slot={() => 'useCheckoutPricing() return value'} />
+
```

Comment:
I'm thinking a brief description here will help provide context for those just clicking around, with a link back to the `useCheckoutPricing` section for a full overview.

Reply:
Agreed, good call